src/main/kotlin/io/github/anki/anki/controller/AuthController.kt

```diff
@@ -0,0 +1,69 @@
+package io.github.anki.anki.controller
+
+import io.github.anki.anki.controller.dto.auth.JwtResponseDto
+import io.github.anki.anki.controller.dto.auth.SignInRequestDto
+import io.github.anki.anki.controller.dto.auth.SignUpRequestDto
+import io.github.anki.anki.controller.dto.auth.UserCreatedMessageResponseDto
+import io.github.anki.anki.controller.dto.mapper.toUser
+import io.github.anki.anki.service.UserService
+import io.github.anki.anki.service.model.User
+import io.github.anki.anki.service.model.mapper.toJwtDto
+import io.github.anki.anki.service.secure.SecurityService
+import jakarta.validation.Valid
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@CrossOrigin(
+    origins = ["*"],
+    maxAge = AuthController.MAX_AGE.toLong(),
+)
+@RestController
+@RequestMapping(AuthController.BASE_URL)
+class AuthController @Autowired constructor(
+    val userService: UserService,
+    val secureService: SecurityService,
+) {
+
+    @PostMapping(SIGN_IN)
+    @ResponseStatus(HttpStatus.OK)
+    fun authenticateUser(@RequestBody signInRequestDto: @Valid SignInRequestDto?): JwtResponseDto {
+        LOG.info(
+            "IN: ${AuthController::class.java.name}: ${BASE_URL}${SIGN_IN} with userName${signInRequestDto!!.userName}",
+        )
+
+        userService.signIn(signInRequestDto.toUser())
```

Comment:
That's totally insane that you need to call 2 methods from different services to authenticate user.

I suggest move security service to user service and auth user only buy `userService.signIn`